client/src/client_sync/v17/mod.rs

```diff
@@ -246,3 +247,40 @@ pub enum SetBanCommand {
     Add,
     Remove,
 }
+
+/// Args for the `importmulti` method
+#[derive(Clone, Debug, Default, Deserialize, PartialEq, Serialize)]
+pub struct ImportMultiRequest {
+    /// Descriptor to import. If using descriptor, donot also provide address/scriptPubKey, scripts, or pubkeys.
+    #[serde(default, skip_serializing_if = "Option::is_none")]
+    pub desc: Option<String>,
+    /// Type of scriptPubKey (string for script, json for address). Should not be provided if using descriptor.
+    #[serde(rename = "scriptPubKey", skip_serializing_if = "Option::is_none")]
+    pub script_pub_key: Option<ImportMultiScriptPubKey>,
+    /// Creation time of the key expressed in UNIX epoch time, or the string "now" to substitute the current synced blockchain time.
+    pub timestamp: ImportMultiTimestamp,
+}
+
+/// `scriptPubKey` can be a string for script or json for address.
+#[derive(Clone, Debug, Deserialize, PartialEq, Serialize)]
+#[serde(untagged)]
+pub enum ImportMultiScriptPubKey {
+    /// The script.
+    Script(String),
+    /// The address.
+    Address { address: String },
+}
+
+/// `timestamp` can be a number (UNIX epoch time) or the string `"now"`
+#[derive(Clone, Debug, Deserialize, PartialEq, Serialize)]
+#[serde(untagged)]
+pub enum ImportMultiTimestamp {
+    /// The string "now".
+    Now(String),
+    /// The UNIX epoch time.
+    Time(u64),
```

Comment:
```suggestion
    /// The UNIX timestamp.
    Time(u64),
```

'epoch' is the start - time 0